src/main/java/com/redhat/red/build/koji/model/xmlrpc/KojiImageBuildInfo.java

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright (C) 2015 Red Hat, Inc. (jcasey@redhat.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.redhat.red.build.koji.model.xmlrpc;
+
+import org.commonjava.rwx.binding.anno.DataKey;
+import org.commonjava.rwx.binding.anno.StructPart;
+
+@StructPart
+public class KojiImageBuildInfo
```

Comment:
The only thing non-optional appears to be RPM support. The Maven and Windows support can be enabled/disabled via a boolean option in the config.

There are three additional build types currently: maven, image, and win. We already have maven implemented. Therefore, I simply followed the exact setup that we already have for `KojiMavenBuildInfo`. This makes sense as the types are handled the exact same way in the koji code. For example, we have these in the API:

getImageArchive(archive_id, strict=False)
getImageBuild(buildInfo, strict=False)
getMavenArchive(archive_id, strict=False)
getMavenBuild(buildInfo, strict=False)
getWinArchive(archive_id, strict=False)
getWinBuild(buildInfo, strict=False)`

So, I just followed the lead of `getMavenBuild` and `KojiMavenBuildInfo` on this and I guess I am not seeing what, if anything, makes them any different from maven.